ui/src/containers/SortDropDown/index.tsx

```diff
@@ -1,15 +1,14 @@
 import React, { useState } from 'react';
 import { Select, SelectVariant, SelectOption, SelectOptionObject } from '@patternfly/react-core';
-import { useObserver } from 'mobx-react';
+import { observer } from 'mobx-react';
 import { useMst } from '../../store/root';
 import { SortByFields } from '../../store/resource';
 import './SortDropDown.css';
 
-const Sort: React.FC = () => {
+const Sort: React.FC = observer(() => {
```

Comment:
Why this change ?? Is it not working if we use `useObserver` in return which renders as the state updates